### PR TITLE
chore(amazonq): update user-service-2.json

### PR DESCRIPTION
**Pull request by leigaol**
## Solution

---

License: I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by fangwade007 — APPROVED** [review]

**Comment by justinmk3** [comment]
ci is failing

File: packages/core/src/codewhisperer/client/user-service-2.json [hunk 1]
```diff
@@ -18,13 +18,25 @@
...
+                },
+                {
+                    "shape": "InternalServerException"
+                },
+                {
+                    "shape": "ValidationException"
+                },
+                {
+                    "shape": "AccessDeniedException"
+                }
```
Comment (Will-ShaoHua) [inline packages/core/src/codewhisperer/client/user-service-2.json:39]: can we use the file with the same formatting?

Comment (Will-ShaoHua) [reply packages/core/src/codewhisperer/client/user-service-2.json:39]: diff is too large and most of them are formatting changes

Comment (Will-ShaoHua) [reply packages/core/src/codewhisperer/client/user-service-2.json:39]: hardly can tell which changes are the purpose of this pr

**Comment by Will-ShaoHua** [comment]
#5695

**Pull request closed without merging**